### DISCUSSION TITLE
Looking For a Vuetify UI Kit

### DISCUSSION THREAD

**Discussion by lartheon** [vuetifyjs/vuetify#12361] [Q&A] [answered]
Hi, I've used Adobe XD  for prototyping on previous projects, I would like to mock up some designs for our Vuetify website but I've not been able to find any Vuetify UI Kits I can template from. 

Does anyone know of any?

Thanks

**Comment by johnleider** [comment]
Any Material Design kit should get your feet well into the door. I'm aware of some that are in development specifically for Vuetify, but nothing that is public at the moment. No eta on that either.

**Reply by itamar-inbar** [reply]
Hi, any news about Vuetify kit for designers? 

Thanks

**Reply by johnleider** [reply]
This is being built right now. It will be available relatively soon.

**Reply by sl0wik** [reply]
Any chance to access to beta version?

**Reply by sl0wik** [reply]
The kit is available here https://store.vuetifyjs.com/products/vuetify-ui-kit-adobe-xd

**Comment by johnleider** [comment, marked as answer]
We now have Figma and Adobe XD kits with Affinity and Sketch coming soon.

https://store.vuetifyjs.com/collections/ui-kits

**Comment by estudiooka** [comment]
https://www.vuetifyforfigma.com
